src/dispatch/static/dispatch/src/service_feedback/Table.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <v-container fluid>
+    <delete-dialog />
+    <v-row no-gutters>
+      <v-col>
+        <div class="headline">Service feedback</div>
+      </v-col>
+      <v-col class="text-right">
+        <table-filter-dialog :projects="defaultUserProjects" />
+      </v-col>
+    </v-row>
+    <v-row no-gutters>
+      <v-col>
+        <v-card elevation="0">
+          <v-card-title>
+            <v-text-field
+              v-model="q"
+              append-icon="search"
+              label="Search"
+              single-line
+              hide-details
+              clearable
+            />
+          </v-card-title>
+          <v-data-table
+            :headers="headers"
+            :items="items"
+            :server-items-length="total"
+            :page.sync="page"
+            :items-per-page.sync="itemsPerPage"
+            :sort-by.sync="sortBy"
+            :sort-desc.sync="descending"
+            :loading="loading"
+            loading-text="Loading... Please wait"
+          >
+            <template #item.individual="{ item }">
+              <participant
+                v-if="item.individual"
```

Comment:
We should populate this item in this column as "Anonymous" if the participant submitted their feedback anonymously.

Reply:
Good idea. Added!